src/app/components/publiclists/teamlist/teamcard/teamcard.component.html

```diff
@@ -1,41 +1,41 @@
 <div class="team-card-container" [class.selected]='isSelected' [class.noexpand] = 'isHost' [class.expand] = '!isHost'>
 <div class="team-card-container-child">
-  <div class="team-content" (click)='selectToggle()'>
-    <i class="fa fa-circle selectstate" *ngIf='!isSelected && (isHost || isOnChallengePage)'></i>
-    <i class="fa fa-check selectstate" aria-hidden="true" *ngIf='isSelected && (isHost || isOnChallengePage)'></i>
-    <i class="fa fa-users" aria-hidden="true"></i>
-    <div class="team-details">
-      <span class="title toptitle">
-	    Team:
-      </span>
-      <span *ngIf="teamView['team_url']">
-      	<a class="orange-text" target="_blank" href="{{teamView['team_url']}}">{{teamView['team_name']}}</a>
-      </span>
-      <span *ngIf="!teamView['team_url']">
-	    {{teamView['team_name']}}
-      </span>
-      <br>
-      <span class="title">
-	    Created by:
-      </span>
-      <span>
-	    {{teamView['created_by']}}
-      </span>
+    <div class="team-content" (click)='selectToggle()'>
```

Comment:
Hey @Suryansh5545 I think this is causing the problem. You need to add this `on click` excluding the radio inputs.

Reply:
i cant understand sir do i have to add this to my radio button?

Reply:
Nope, not on to your radio buttons but excluding them.

Reply:
So the function shouldn't apply to radio button?

Reply:
Nope, not on to your radio buttons but excluding them.

Reply:
@Sanji515 Sir like i have created two function one is executed when the team-content is pressed which will make it open and close and other function executes when the radio button is pressed so the team can be selected .

Reply:
> Nope, not on to your radio buttons but excluding them.

I'm saying this cause in `selectToggle` function you are just toggling the selection class but not calling `selectTeam` function.

But now I think we don't need to do that, you just need to remove the extra radio type input and its condition [here](https://github.com/Cloud-CV/EvalAI-ngx/pull/276/files#diff-1343c36995ebad881639aeba7d3a10e3R4)

Reply:
@Sanji515  I have done the changes sir.